### PR TITLE
Don't generate raw html in notebooks

### PR DESCRIPTION
**Pull request by amueller**
Hugo doesn't allow raw html.

### CODE REVIEW

**Comment by amueller** [comment]
accidentally layered this on top of #29 :-/

File: website/jupyter_nbconvert_config.py [hunk 1]
```diff
@@ -0,0 +1,661 @@
...
+#
+#  Useful for display data priority that might be used by many transformers
+
+## Deprecated default highlight language as of 5.0, please use language_info
+#  metadata instead
+#c.NbConvertBase.default_language = 'ipython'
+
+## An ordered list of preferred output type, the first encountered will usually
+#  be used when converting discarding the others.
+c.NbConvertBase.display_data_priority = ['image/png', 'image/jpeg', 'text/markdown', 'text/plain', 'text/html', 'application/pdf', 'text/latex', 'image/svg+xml']
```
Comment (bpkroth) [inline website/jupyter_nbconvert_config.py:94]: For the record: usually text/markdown and text/plain are listed before the image types

Comment (amueller) [reply website/jupyter_nbconvert_config.py:94]: todo: add a comment with the default value above.

Comment (amueller) [reply website/jupyter_nbconvert_config.py]: done

**Review by bpkroth — APPROVED** [review]
LGTM

**Pull request merged**